sapl/materia/models.py

```diff
@@ -262,11 +262,11 @@ def data_entrada_protocolo(self):
         from sapl.protocoloadm.models import Protocolo
         if self.ano and self.numero_protocolo:
             try:
-                protocolo = Protocolo.objects.get(
+                protocolo = Protocolo.objects.filter(
                     ano=self.ano,
                     numero=self.numero_protocolo)
-                if protocolo.timestamp:
-                    return protocolo.timestamp.date()
+                if protocolo.first().timestamp:
+                    return protocolo.first().timestamp.date()
                 else:
                     return protocolo.data
             except ObjectDoesNotExist:
```

Comment:
Com a mudança aqui do ```.get``` para o ```.filter``` não é mais necessário esse ```try... except``` acho que poderia mudar para algo assim:
```suggestion
            if not protocolo:
                return ''
```

Reply:
+1